### ISSUE TITLE
Model loading/training error

### ISSUE THREAD

**Issue by lvaleriu** [dhlab-epfl/dhSegment#22]
When executing the following command: python train.py with demo/demo_config.json
I get this error. FYI I've followed the installation instructions with conda.

`InternalError (see above for traceback): cuDNN launch failure : input shape([1,3,1095,538]) filter shape([7,7,3,64])
         [[{{node resnet_v1_50/conv1/Conv2D}} = Conv2D[T=DT_FLOAT, data_format="NCHW", dilations=[1, 1, 1, 1], padding="VALID", strides=[1, 1, 2, 2], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:GPU:0"](gradients/resnet_v1_50/conv1/Conv2D_grad/Conv2DBackpropFilter-0-TransposeNHWCToNCHW-LayoutOptimizer, resnet_v1_50/conv1/weights/read)]]`

**Comment by solivr** [comment]
This looks like you need to upgrade your cuDNN library or downgrade your tensorflow library. I would recommend changing tensorflow to v1.11 or v1.10.

**Comment by lvaleriu** [comment]
Ok. I’ll try that and come back to you. Thanks!

**Comment by solivr** [comment]
Closing because of inactivity.
